### PR TITLE
Add slack as an option for WebHooks

### PR DESCRIPTION
**Pull request by christopheranderson**

### CODE REVIEW

**Comment by mathewc** [comment]
Related to https://github.com/Azure/azure-webjobs-sdk-script/issues/318. Can we close that one now?

File: Bindings/bindings.json [hunk 1]
```diff
@@ -717,7 +717,10 @@
                         {
                             "value": "genericJson",
                             "display": "Generic JSON"
-                        }
+                        },
+                        {
+                            "value":"slack",
+                            "display":"Slack"
```
Comment (mathewc) [inline Bindings/bindings.json]: This "display" will be the label shown on the Secret shown in the UI, correct?

**Comment by mathewc** [comment]
:ship:

**Comment by soninaren** [comment]
Does this need to go 0.8

**Comment by soninaren** [comment]
Manually checking this in

**Pull request closed without merging**